src/components/Loader.tsx

```diff
@@ -0,0 +1,17 @@
+import useVictim from '../hooks/useVictim';
+
+export const Loader = () => {
+  const { isLoading, error } = useVictim();
+
+  return (
+    <div className='spinner'>
+      {isLoading && <img src='/spinner2.gif' height='450' />}
+
+      {error &&
+        <div className='error'>
+          <h1>je to v piči</h1>
+          <p className='error-message'>{error}</p>
+        </div>}
+    </div>
+  );
+};
```

Comment:
something named `Loader` probably shouldn't handle errors.

The way I would go about this is renaming this component to something like `DataLoaderBoundary` and having it accept `children` prop which would be rendered if load was successful (otherwise it would behave as it does now)

*(This way you'd avoid consuming the loading state and error in 2 places)*